### PR TITLE
Make VoiceChrome apply dark mode when system configuration change dyn…

### PR DESCRIPTION
**Pull request by Yuseon-Han**
…amically.

for this, ChromeWindowContentLayout changed to BottomSheet Itself to observing configuration change.

plus) view elevation attribute replaced by using drawable xml because 'elevation' attribute has dependent with api version.

### CODE REVIEW

File: nugu-ux-kit/src/main/res/drawable-xxxhdpi/nugu_view_chrome_window_dark_background.xml [hunk 1]
```diff
@@ -1,5 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
 <layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:bottom="50dp">
```
Comment (shshshl09) [inline nugu-ux-kit/src/main/res/drawable-xxxhdpi/nugu_view_chrome_window_dark_background.xml:3]: same with 'nugu_view_chrome_window_light_background'?

Comment (Yuseon-Han) [reply nugu-ux-kit/src/main/res/drawable-xxxhdpi/nugu_view_chrome_window_dark_background.xml:3]: yes it is

**Review by shshshl09 — APPROVED** [review]

File: nugu-ux-kit/src/main/java/com/skt/nugu/sdk/platform/android/ux/widget/ChromeWindowContentLayout.kt [hunk 2]
```diff
@@ -106,13 +104,20 @@ class ChromeWindowContentLayout @JvmOverloads constructor(
...
             chipsView = findViewById(R.id.chipsView)
-            bottomSheet = findViewById(R.id.fl_bottom_sheet)
             sttTextView = findViewById(R.id.tv_stt)
             voiceChrome = findViewById(R.id.voice_chrome)
+
         }
+        view.addView(this)
+        with((this.layoutParams as CoordinatorLayout.LayoutParams)) {
+            behavior = ChromeWindowBottomSheetBehavior<FrameLayout>(context, null)
+            height = dpToPx(78f, context)  // bottomSheet height(68dp) + shadow height(10dp)
```
Comment (tommy5oo) [inline nugu-ux-kit/src/main/java/com/skt/nugu/sdk/platform/android/ux/widget/ChromeWindowContentLayout.kt:116]: Missing width attributes

Comment (Yuseon-Han) [reply nugu-ux-kit/src/main/java/com/skt/nugu/sdk/platform/android/ux/widget/ChromeWindowContentLayout.kt:118]: I added it. thanks

**Review by tommy5oo — APPROVED** [review]

**Pull request merged**